authentik/stages/redirect/models.py

```diff
@@ -0,0 +1,49 @@
+"""authentik redirect stage"""
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from django.views import View
+from rest_framework.serializers import BaseSerializer
+
+from authentik.flows.models import Flow, Stage
+
+
+class RedirectMode(models.TextChoices):
+    """Mode a Redirect stage can operate in"""
+
+    STATIC = "static"
+    FLOW = "flow"
+
+
+class RedirectStage(Stage):
+    """Redirect the user to another flow, potentially with all gathered context"""
+
+    keep_context = models.BooleanField(default=True)
+    mode = models.TextField(choices=RedirectMode.choices, default=RedirectMode.STATIC)
```

Comment:
```suggestion
    mode = models.TextField(choices=RedirectMode.choices, default=RedirectMode.STATIC)
```
I wouldn't set a default here

Reply:
Fair enough. On the frontend it's fine, but there's no need here. Removed.